HelpSource/Guides/OSC_communication.schelp

```diff
@@ -73,3 +73,66 @@ thisProcess.addOSCRecvFunc(f);
 // stop posting.
 thisProcess.removeOSCRecvFunc(f);
 ::
+
+section::OSC type tags and sclang
+
+With an exception, sclang will only send the four basic OSC type tags. sclang understands seven additional nonstandard type tags. Inbound OSC messages must have type tags, or an error will be thrown.
+
+The following conversions are supported outbound:
+
+list::
+## Integers become "i". Both OSC and sclang use 32-bit signed integers.
+## Booleans become "i", with true translating to 1 and false translating to 0.
+## Nil becomes "i", translating to value 0.
+## Floats become "f" (32-bit float). sclang's Floats are 64-bit, so the conversion to OSC is lossy.
+## Strings and Symbols become "s" (string).
+## Int8Arrays become "b" (blob).
+## Chars are added as a raw type tag, appending no data to the OSC packet.
+::
+
+If code::NetAddr.useDoubles:: is set to true, then in outbound messages, sclang will use the nonstandard "d" (double) type tag instead of "f". OSC doubles are 64-bit.
+
+warning::You shouldn't send a raw type tag unless the destination agrees that the character is supposed to consume zero bytes in the packet data (as in N/I/T/F), otherwise following arguments will be corrupted.
+
+In this example, sclang sends a message to itself containing a raw "i" type tag. Outbound, the raw type tag appends zero bytes, but inbound, "i" consumes four bytes. The "hell" in "hello" is interpreted as an integer, and the string starts at "o".
+
+code::
+(
+OSCFunc(_.postln, \test).oneShot;
+NetAddr.localAddr.sendMsg(\test, $i, "hello world");
+// -> [ /test, 1751477356, o world ]
+)
+::
+
+In general, raw type tags should be avoided.
+::
+
+The same four standard type tags are supported inbound:
+
+list::
+## "i" (32-bit signed integer) becomes Integer
+## "f" (32-bit float) becomes Float
+## "s" (string) becomes Symbol
+## "b" (blob) becomes Int8Array
+::
+
+The following nonstandard type tags are supported inbound:
+
+list::
+## "d" (64-bit float) becomes Float
+## "m" (4-byte MIDI message) becomes Int8Array
+## "c" (character) becomes Char
+## "t" (time tag) becomes ???
+## "T" (true) and "F" (false) become Boolean
+## "I" (infinity) becomes Float
+## "N" (nil) becomes nil
+::
+
+The following nonstandard type tags are NOT supported:
+
+list::
+## "r" (RGBA color)
+## "[" and "]" (array start and end)
```

Comment:
Maybe we should add that there exist methods to convert outbound messages to use these array delimiters (`asOSCArgArray`, usually by ways of `asControlInput`). They are used internally when sclang sends messages to scynth.

For inbound ones, we use `unfoldOSCArrays`, but they are still here: https://github.com/supercollider-quarks/OpenObject/blob/master/classes/extOSCArgs.sc

It might be good if those methods were built in.